### PR TITLE
[CSS] cluster create backup strategy fix

### PR DESCRIPTION
**Pull request by vladimirvshivkov**

### CODE REVIEW

**Review by YustinaKvr — DISMISSED** [review]

File: tests/integration/targets/dms/aliases [hunk 1]
```diff
@@ -0,0 +1 @@
+disabled
```
Comment (anton-sidelnikov) [inline tests/integration/targets/dms/aliases:1]: Why disabled?

File: plugins/modules/css_cluster.py [hunk 2]
```diff
@@ -225,14 +252,20 @@ class CssClusterModule(OTCModule):
...
         net=dict(type='str'),
         security_group=dict(type='str'),
         tags=dict(required=False, type='list', elements='dict'),
-        backup_period=dict(type='str'),
-        backup_prefix=dict(type='str'),
-        backup_keepday=dict(type='int'),
+        backup_strategy=dict(type='dict', options=dict(
+            period=dict(type='str'),
+            prefix=dict(type='str'),
+            keepday=dict(type='int'),
```
Comment (anton-sidelnikov) [inline plugins/modules/css_cluster.py:263]: keepday has a range of values, maybe check it inside code?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: tests/integration/targets/dms/aliases (hunk 1 above)
Comment (vladimirvshivkov) [reply tests/integration/targets/dms/aliases:1]: @vladimirvshivkov
[Service DMS is currently unavailable](https://github.com/opentelekomcloud/ansible-collection-cloud/pull/218/commits/e4d3d4248162f1583b5938f29cc95cc2cd7a13e1)

Comment (vladimirvshivkov) [reply tests/integration/targets/dms/aliases:1]: DMS service is shut down

File: plugins/modules/css_cluster.py (hunk 2 above)
Comment (vladimirvshivkov) [reply plugins/modules/css_cluster.py:263]: ok

File: plugins/modules/css_cluster.py [hunk 3]
```diff
@@ -327,13 +356,29 @@ def run(self):
...
+                        attrs['backupStrategy']['period'] = self.params['backup_strategy']['period']
+                    if self.params['backup_strategy']['prefix']:
+                        attrs['backupStrategy']['prefix'] = self.params['backup_strategy']['prefix']
+                    if self.params['backup_strategy']['bucket']:
+                        attrs['backupStrategy']['bucket'] = self.params['backup_strategy']['bucket']
+                    if self.params['backup_strategy']['basepath']:
+                        attrs['backupStrategy']['basePath'] = self.params['backup_strategy']['basepath']
+                    if self.params['backup_strategy']['agency']:
+                        attrs['backupStrategy']['agency'] = self.params['backup_strategy']['agency']
+                    if self.params['backup_strategy']['keepday'] in range(1, 90):
```
Comment (anton-sidelnikov) [inline plugins/modules/css_cluster.py:371]: is it possible to set more that 90 if you, check the range inside parameters section?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/css_cluster.py (hunk 3 above)
Comment (vladimirvshivkov) [reply plugins/modules/css_cluster.py:371]: checking range inside argument spec is not working

Comment (vladimirvshivkov) [reply plugins/modules/css_cluster.py:371]: it just passing through

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by vladimirvshivkov** [comment]
recheck

**Review by Polina-Gubina — APPROVED** [review]

**Pull request merged**